src/Library.java

```diff
@@ -0,0 +1,186 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Library {
+    private Book[] books = new Book[100];
+    private int bookCount;
+    private Student[] students = new Student[100];
+    private int studentCount;
+
+    private static final String CYAN = "\u001B[36m";
+
+    /**
+     * Adds a new book to the library itself. (Book array)
+     * @param title
+     * @param author
+     * @param isbn
+     */
+    public void addBook(String title, String author, long isbn) {
+        if (bookCount < books.length) {
+            books[bookCount++] = new Book(title, author, isbn);
+        } else {
+            System.out.println("Library is full. Cannot add more books.");
```

Comment:
ideally, the actual interface with the terminal should not be here - it should be inside the controller method that calls it. You could have this return a boolean, and false means "full" and have it be interpreted at the top. Or you could create a custom exception...